.github/workflows/linting-and-tests.yml

```diff
@@ -263,57 +208,45 @@ jobs:
           --health-timeout 5s
           --health-retries 5
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
+      - name: Checkout project
+        uses: actions/checkout@v3
+      - name: Setup Python
+        uses: ./.github/actions/setup-python
         with:
-          python-version: "3.11.4"
-          cache: "pip"
-          cache-dependency-path: |
-            engine/requirements.txt
-            engine/requirements-dev.txt
+          python-requirements-paths: "engine/requirements.txt engine/requirements-dev.txt"
```

Comment:
If this is the default value, is it needed here? (same below)

Reply:
good catch, thanks 👍